src/util.cpp

```diff
@@ -1436,6 +1436,31 @@ bool CLocale::IsCountryCodeSupported ( unsigned short iCountryCode )
 #endif
 }
 
+QLocale::Country CLocale::GetCountryCodeByTwoLetterCode ( QString sTwoLetterCode )
+{
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 2, 0 )
+    return QLocale::codeToTerritory ( sTwoLetterCode );
```

Comment:
Relying on the compiler to optimise this "call" to inline?  Or have the .h declare inline the 6.2.0 version?

Reply:
This function is called once on server start. I would rely on the compiler to do its thing... :)